test/helpers/constants.ts

```diff
@@ -77,6 +77,9 @@ export const RUNTIME_CONSTANTS = {
     EXTRINSIC_GAS_LIMIT: 52_000_000n,
     // Maximum Gas to PoV ratio used in the gasometer
     GAS_PER_POV_BYTES: 16n,
+    // Storage read/write costs
+    STORAGE_READ_COST: 41_742_000n,
+    STORAGE_READ_GAS_COST: 1669n,
```

Comment:
I am not sure about this value. Where did you extract it?

Reply:
It is mainly based on the changes to the rust tests on #2786, which I might be misreading. There, several precompiles tests changed their estimated gas cost from 1000 to 1669 and other worked with doubled values.

Will do some more research on them to see if we can keep the extracted constant (even if the value needs to change), but we could remove it for the purpose of this PR and only have 1 test affected; not that impactful.

Reply:
I think we should only keep `STORAGE_READ_COST` for now since it is a known constant.

Reply:
Giving it a another look, the `1669` updates present in #2786 refer exactly to `STORAGE_READ_COST / WEIGHT_TO_GAS_RATIO` (47_000_000 / 25_000). This confirms it is the DB read cost. I'll update so we can keep `STORAGE_READ_GAS_COST` and use it in a way that it is relative to the `STORAGE_READ_COST`.